app/lib/ui/views/signin/sign_in_screen.dart

```diff
@@ -88,17 +120,26 @@ class SignInScreen extends StatelessWidget {
                   SizedBox(
                     height: 100.0,
                   ),
+                  TextButton(
+                    child: Text('Do not have an account? Sign Up Here'),
+                    onPressed: () => widget._navigationService.navigateTo(SignUpScreen.route),
```

Comment:
I think to solve my problem in a previous comment, we should do `navigationService.navigateBackTo(homescreen)` then `navServ.push(login)`. Cleaner navigation than causing a loop between welcome, login, and sign up.

- [ ] Do pop until, then push

I can also do this change in #162 .

Reply:
This is part of #162, don't do this.